additional references to samples and SDK

SychevIgor opened the pull request:
additional references to samples and SDK for Cognitive Service Speech

azurecla commented:
Hi __@v-micada__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-micada). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
adding test topic

Commit:
fixing tab groups

Commit:
Maintenance and updates

Commit:
fixing toc

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into maintenance-updates

Commit:
drafting

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into maintenance-updates

Commit:
drafting

Commit:
drafting

Commit:
fixing merge

Commit:
getting rid of line breaks

Commit:
drafting

Commit:
drafting

SychevIgor committed:
addtional references to samples and SDK

addtional references to samples and SDK for Cognitive Service Speech

burkeholland committed:
Node.js Samples - Postgres - Delete Data updates

burkeholland committed:
Update connect-nodejs.md

Commit:
Merge branch 'cfowler-tabbed-conceptual-test' of https://github.com/SyntaxC4-MSFT/azure-docs-pr

Commit:
moving things around a little

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

burkeholland committed:
Update connect-nodejs.md

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into maintenance-updates

Commit:
fixing wording

burkeholland committed:
Update connect-nodejs.md

Commit:
words

burkeholland committed:
Update connect-nodejs.md

burkeholland committed:
Update connect-nodejs.md

Commit:
removing hard dates per Ziv

Commit:
safekeeping

burkeholland committed:
Update delete snippet

Modify disclaimer and bring `database` property inline with consistent naming conventions.

burkeholland committed:
Update connect-nodejs.md

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
removing how to from the conceptual pr

Commit:
linux side

tonguyen10 committed:
Delete 5.png

tonguyen10 committed:
Add files via upload

tonguyen10 committed:
Create billing-getting-started2.md

tonguyen10 committed:
Update billing-getting-started2.md

tonguyen10 committed:
Delete cost-analysis.PNG

tonguyen10 committed:
Add files via upload

tonguyen10 committed:
Delete pricing-calc.PNG

tonguyen10 committed:
Add files via upload

tonguyen10 committed:
Update billing-getting-started2.md

tonguyen10 committed:
Delete cost-analysis.PNG

tonguyen10 committed:
Delete pricing-calc.png

tonguyen10 committed:
Delete pricing-calcVM.png

tonguyen10 committed:
Add files via upload

tonguyen10 committed:
Update billing-getting-started2.md

tonguyen10 committed:
Add files via upload

tonguyen10 committed:
Delete spend-history.PNG

tonguyen10 committed:
Delete costhistory.PNG

tonguyen10 committed:
Add files via upload

tonguyen10 committed:
Update billing-getting-started2.md

Commit:
making include and fixing Linux TOC

Commit:
fixing links

SaeedAkhter-MSFT committed:
Update B2C FAQ - Dynamics Portal integration is now available

Commit:
acrolinx

rwike77 committed:
added file

rwike77 committed:
updates

Commit:
Updated with correct limit

heyyjudes committed:
Inserting change feed into scenarios

heyyjudes committed:
Updating with change feed examples linked

heyyjudes committed:
adding change feed to document db intro

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into maintenance-updates

Commit:
fixing link

mumian committed:
refresh articles

mumian committed:
fix a link

mumian committed:
acrolinx

rwike77 committed:
Updated script table

Commit:
assigning the articles I am going to delete, to myself

Commit:
deleting articles and creating redirects

MarkMcGeeAtAquent committed:
Cosmos DB: tutorial updates

MarkMcGeeAtAquent committed:
Delete nosql-tutorial-account-database.png

MarkMcGeeAtAquent committed:
Delete nosql-tutorial-manage-nuget-pacakges-2.png

MarkMcGeeAtAquent committed:
Delete nosql-tutorial-new-project-2.png

MarkMcGeeAtAquent committed:
Cosmos DB: updated art

mumian committed:
update

gsacavdm committed:
Add guidelines for o365 gcc

mumian committed:
editor feedback

gsacavdm committed:
Fix typo

gsacavdm committed:
Few more minor fixes

mumian committed:
format

tonguyen10 committed:
Create billing-getting-started.experimental.md

tonguyen10 committed:
Update billing-getting-started.experimental.md

tonguyen10 committed:
Delete billing-getting-started2.md

tonguyen10 committed:
Update billing-getting-started.md

tonguyen10 committed:
Update billing-getting-started.md

tonguyen10 committed:
Update billing-getting-started.md

chetansriv committed:
1058767: SaaS App Tutorial: Merchlogix update

rwike77 committed:
added lb script

MarkMcGeeAtAquent committed:
fixed screencap

tonguyen10 committed:
Update billing-getting-started.md

Commit:
fixing redirects

rwike77 committed:
fixed link

rwike77 committed:
updated text

cherylmc committed:
Update vpn-gateway-vpn-faq.md

Commit:
fix

rmca14 committed:
Merge pull request #19502 from MarkMcGeeAtAquent/documentdb-get-started

Cosmos DB: tutorial updates

Commit:
fix

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into basic

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into maintenance-updates

heyyjudes committed:
Fixing spelling mistakes

Commit:
Update analysis-services-overview.md

v-thepet committed:
Merge pull request #19511 from tonguyen10/master

Update cost screenshot and unprevented costs experiment

Commit:
Update analysis-services-compat-level.md

Commit:
metadata fix

v-thepet committed:
Merge pull request #19419 from saveenr/patch-7

Updated with correct limit

Commit:
Merge pull request #19519 from Minewiskan/patch-13

Update analysis-services-overview.md

cherylmc committed:
update

Commit:
Merge pull request #19521 from Minewiskan/patch-14

Update analysis-services-compat-level.md

heyyjudes committed:
Grammar update

cherylmc committed:
update

heyyjudes committed:
Updating typo

rwike77 committed:
minor updates

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
adding warning for FTP disable in Germany regions

MarkMcGeeAtAquent committed:
Merge pull request #6 from MicrosoftDocs/master

update master

johnkemnetz committed:
new partner

Commit:
Merge pull request #19520 from cherylmc/basic

Basic

Commit:
test

Commit:
removing tabbed doc that slipped into branch

johnkemnetz committed:
Update monitoring-partners.md

Commit:
fix

Commit:
moving the warning down a bit

v-thepet committed:
Merge pull request #19509 from rwike77/lbscript

Lb script

SaeedAkhter-MSFT committed:
fix deprecated terms

Commit:
changing warning to important

MarkMcGeeAtAquent committed:
Cosmos DB: tutorial updates

Commit:
fix

Commit:
grr

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into active

Commit:
Merge pull request #19355 from SaeedAkhter-MSFT/faqupdate

Update Azure AD B2C FAQ - Dynamics Portal integration is now available

cherylmc committed:
update format

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into active

ecfan committed:
Post-launch edits per feedback and other tweaks

ecfan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into logic-app-event-grid

v-thepet committed:
Merge pull request #19529 from cynthn/test

Test

cynthn committed:
Merge branch 'master' into maintenance-updates

ecfan committed:
Fix Acrolinx issues

ecfan committed:
Fix Acrolinx issues

cherylmc committed:
update to add new skus

ecfan committed:
Fix Acrolinx issues

Commit:
task 384: add DSA - Adaptive Image Compression , and DSA - Object Prefetch to feature matrix

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-1708

curtand committed:
added note, removed link, etc.

Commit:
task 384: add anchor links

Commit:
Merge pull request #19540 from dksimpson/dean-1708

Dean 1708

Commit:
devicesfaq01

vikramdesai01 committed:
change the last modified

Commit:
devicestoc02

Commit:
Merge pull request #19543 from vikramdesai01/patch-1

change the last modified date

Commit:
Merge pull request #19527 from SyntaxC4-MSFT/cfowler-ftp-disable

Adding Warning that FTP will be disabled in Germany Regions

GitHubber17 committed:
Merge pull request #19528 from johnkemnetz/adding-partner

new partner

GitHubber17 committed:
Merge pull request #19534 from ecfan/logic-app-event-grid

Post-release edits per feedback and other cleanup

GitHubber17 committed:
Merge pull request #17220 from cynthn/maintenance-updates

Maintenance and updates overview - TFS item 1034357 - MVC

gsacavdm committed:
Tweaks to az-gov-dev

GitHubber17 committed:
Merge pull request #19542 from MarkusVi/devicesfaq01

devicesfaq01

GitHubber17 committed:
Merge pull request #19505 from gsacavdm/o365gcc

Add dev guidance for Azure Gov + O365 GCC

pvrk committed:
Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

Cleaning up the file as per suggestions from doc admins

mumian committed:
format

mumian committed:
format list

mumian committed:
format

mumian committed:
format list

mumian committed:
format

Commit:
ECMA2116040_Augupdate

rjagiewich committed:
Merge pull request #19544 from MarkusVi/devicestoc02

devicestoc02

rjagiewich committed:
Merge pull request #19477 from mumian/stale01-0815

refresh articles

Commit:
Merge pull request #19553 from fimguy/ECMA2116040_Augupdate

ECMA2116040_Augupdate

Commit:
Merge pull request #19435 from heyyjudes/master

Adding change feed to various documentation pages

rmca14 committed:
Merge pull request #19552 from pvrk/patch-13

Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

Commit:
Merge pull request #19516 from cherylmc/patch-1

Update link vpn gateway faq

Commit:
Merge pull request #19533 from cherylmc/active

Active

rjagiewich committed:
Merge pull request #18821 from burkeholland/patch-18

Node.js Samples - Postgres - Delete Data updates

v-thepet committed:
Merge pull request #19510 from chetansriv/merchlogix

1058767: SaaS App Tutorial: Merchlogix update

v-thepet committed:
Merge pull request #19531 from MarkMcGeeAtAquent/spark-connector

Cosmos DB: Spark Connector tutorial updates

Commit:
Merge pull request #19539 from curtand/class0815

added note, removed link, etc.
